### PR TITLE
Migrates Varscan from devteam and adds new tools to it

### PR DESCRIPTION
**Pull request by bgruening**
I started to migrate the old tool, but this will not be an update of the old tool but rather deprecates it in favor of this tool suite. This is mainly because I could not figure out how to keep the old one but still maintain it with demultiplexing a suite.

### CODE REVIEW

**Comment by bgruening** [comment]
@erasche ready for review.

File: tools/varscan/varscan_copynumber.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+            --p-value ${p_value}
+            --data-ratio ${data_ratio}
+
+    ]]></command>
+
+    <inputs>
+        <param name="normal_pileup" format="pileup" type="data" label="Samtools pileup dataset" help=""/>
+        <param name="tumor_pileup" format="pileup" type="data" label="Samtools pileup dataset" help=""/>
+
+        <param argument="--min-coverage" name="min_coverage" type="integer" value="20" min="1" max="200"
```
[the remark this answers is not in the document]
Comment (bgruening) [reply tools/varscan/varscan_copynumber.xml]: to many changes, like values, desc....

File: tools/varscan/.shed.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+categories:
+  - Variant Analysis
+description: VarScan is a variant caller for high-throughput sequencing data
+homepage_url: https://dkoboldt.github.io/varscan/
+long_description: |
+  VarScan is a variant caller for high-throughput sequencing data.
+name: varscan
+owner: iuc
```
Comment (bgruening) [inline tools/varscan/.shed.yml]: I changes this here to IUC. So the devteam one needs to be depreacted ideally.

**Comment by bgruening** [comment]
Thanks for the review, I hope I got all of them.

File: tools/varscan/.shed.yml (hunk 2 above)
Comment (martenson) [reply tools/varscan/.shed.yml]: easy

File: tools/varscan/varscan_copynumber.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (erasche) [reply tools/varscan/varscan_copynumber.xml]: Sure, that's fine.

**Comment by erasche** [comment]
Looks good!

**Pull request merged**